automated/linux/ltp/ltp.sh

```diff
@@ -31,6 +31,7 @@ SHARD_NUMBER=1
 SHARD_INDEX=1
 
 RUNNER=""
+KIRK_WORKERS=1
```

Comment:
Just a thought. Some nproc auto detection here might be useful as well being able to define a number